arrows/core/close_loops_keyframe.cxx

```diff
@@ -272,14 +272,14 @@ ::stitch( vital::frame_id_t frame_number,
     all_matches[*f] = pool.enqueue(match_func, *f);
   }
   // stitch with all previous keyframes
-  for(auto kitr = keyframes.rbegin(); kitr != keyframes.rend(); ++kitr)
```

Comment:
Given how far apart setting it is to its use, I would have left these alone and instead renamed the first `kitr` to something more meaningful. (Maybe @mleotta has a naming suggestion?)